buildSrc/build.gradle.kts

```diff
@@ -21,25 +21,27 @@ repositories {
 sourceSets.main {
 }
 
+val ktor_version = "2.3.0"
+
 dependencies {
     implementation(kotlin("gradle-plugin", "1.8.10"))
     implementation(kotlin("serialization", "1.8.10"))
 
     val ktlint_version = libs.versions.ktlint.version.get()
     implementation("org.jmailen.gradle:kotlinter-gradle:$ktlint_version")
 
-    implementation("io.ktor:ktor-server-default-headers:2.0.2")
-    implementation("io.ktor:ktor-server-netty:2.0.2")
-    implementation("io.ktor:ktor-server-cio:2.0.2")
-    implementation("io.ktor:ktor-server-jetty:2.0.2")
-    implementation("io.ktor:ktor-server-websockets:2.0.2")
-    implementation("io.ktor:ktor-server-auth:2.0.2")
-    implementation("io.ktor:ktor-server-caching-headers:2.0.2")
-    implementation("io.ktor:ktor-server-conditional-headers:2.0.2")
-    implementation("io.ktor:ktor-server-compression:2.0.2")
-    implementation("io.ktor:ktor-server-content-negotiation:2.0.2")
-    implementation("io.ktor:ktor-serialization-kotlinx:2.0.2")
-    implementation("io.ktor:ktor-network-tls-certificates:2.0.2")
+    implementation("io.ktor:ktor-server-default-headers:$ktor_version")
```

Comment:
Can we use `ktor-bom` here?

Reply:
we can, but I'm not sure why it's needed here